lib/game/platform_manager.dart

```diff
@@ -70,9 +70,9 @@ class PlatformManager extends Component with HasGameRef<DoodleDash> {
     var currentY =
         gameRef.size.y - (random.nextInt(gameRef.size.y.floor()) / 3) - 50;
 
-    // Generate 10 Platforms at random x, y positions and add to list of platforms
+    // Generate 30 Platforms at random x, y positions and add to list of platforms
     // to be populated in the game.
-    for (var i = 0; i < 9; i++) {
+    for (var i = 0; i < 29; i++) {
```

Comment:
increased the number because that's what doodle jump does, I'm not married to this number.